packages/component/src/Localization/ru-RU.js

```diff
@@ -1,16 +1,48 @@
+function xMinutesAgo(date) {
+  const now = Date.now();
+  const deltaInMs = now - new Date(date).getTime();
+  const deltaInMinutes = Math.floor(deltaInMs / 60000);
+  const deltaInHours = Math.floor(deltaInMs / 3600000);
+
+  if (deltaInMinutes < 1) {
+    return 'Сейчас';
+  } else if (deltaInMinutes === 1) {
+    return 'Минуту назад';
+  } else if (deltaInHours < 1 && deltaInMinutes < 5) {
+    return `${ deltaInMinutes } минуты назад`;
+  } else if (deltaInHours < 1 && deltaInMinutes >= 5) {
+    return `${ deltaInMinutes } минут назад`;
+  } else if (deltaInHours === 1) {
+    return 'Час назад';
+  } else if (deltaInHours < 5) {
+    return `${ deltaInHours } часа назад`;
+  } else if (deltaInHours <= 24) {
+    return 'Сегодня';
+  } else if (deltaInHours <= 48) {
+    return 'Вчера';
+  } else if (deltaInHours <= 72) {
+    return 'Позавчера';
+  } else {
+    return new Intl.DateTimeFormat('ru-RU').format(date);
```

Comment:
If the browser doesn't support `Intl`, can you add code to fallback to simple format (although it might be not-quite-working), better than error out.